### ISSUE TITLE
Serve Readme.md when in "depot" mode, so one can load the extension in depot mode from local url

### ISSUE THREAD

**Issue by Stvad** [dvargas92495/roamjs-scripts#8]
It's one of the requirements to be able to serve the "depot" extension from URL: https://roamresearch.com/#/app/developer-documentation/page/n8ECay0Wl

**Comment by dvargas92495** [comment]
All the RoamJS extensions now use `@samepage/scripts` instead of this package, though I'm happy to accept contributions to cover use cases this one is still missing

**Comment by Stvad** [comment]
Is that package opensource/meant for others to use? I'm happy to switch over to that if that's basically a better maintained version of this package 🙂

**Comment by Stvad** [comment]
Also generally curious what's your dev iteration process for depot extensions. What's the fastest way to go from making a change to seeing an update version of extension in roam?

**Comment by dvargas92495** [comment]
yup, https://github.com/samepage-network/samepage.network/tree/main/package/scripts

for Roam extensions, you'll need to add some config to the `package.json`. If you're on the latest version of `roamjs-components`, it should already have `@samepage/scripts` and you'll just need:

```
"samepage": {
  "extends": "node_modules/roamjs-components/package.json"
}
```

If you don't use `roamjs-components`, this is the recommended config for Roam extensions: https://github.com/dvargas92495/roamjs-components/blob/main/package.json#L80-L110

> Also generally curious what's your dev iteration process for depot extensions.

I then just use `samepage dev` and use load from local. 

> What's the fastest way to go from making a change to seeing an update version of extension in roam?

`samepage build` publishes the extension to `samepage.network/releases/*` so that I could load from URL. We may need to add a param to support deploying to a custom location of your choosing, since you won't have the AWS tokens for deploying to samepage's servers
